plugins/woocommerce-admin/client/tasks/reminder-bar/reminder-bar.tsx

```diff
@@ -125,7 +125,14 @@ export const TasksReminderBar: React.FC< ReminderBarProps > = ( {
 		taskListComplete ||
 		reminderBarHidden ||
 		completedTasksCount === 0 ||
-		[ 'Home', 'Shipping', 'Tax', 'Payments' ].includes( pageTitle );
+		[
+			'Home',
+			'Shipping',
+			'Tax',
+			'Payments',
+			'Edit Product',
+			'Add New',
+		].includes( pageTitle );
```

Comment:
Does this still work if page titles are translated?

Reply:
This is a great point @joshuatf , and no, no it doesn't 🤔 . Not quite sure how to deal with this just yet.